pyproject.toml

```diff
@@ -32,6 +32,8 @@ dependencies = [
     "astunparse; python_version<='3.9'",
     "immutabledict",
     "pytools>=2022.1.14",
+    # for dataclass_transform with TypeAlias
```

Comment:
Is this comment accurate? I didn't see any `dataclass_transform` usage?

Reply:
Fixed. That comment didn't make sense. Plus, `dataclass_transform` is now actually used (see below).